rllab/envs/__init__.py

```diff
@@ -0,0 +1,189 @@
+import gym
+from gym.envs.registration import registry, make, spec
+
+
+def register(id, *args, **kvargs):
+    if id in registry.env_specs:
+        return
+    else:
+        return gym.envs.registration.register(id, *args, **kvargs)
+
+
+# ------------bullet-------------
+
+register(
```

Comment:
```python
import pybullet_envs
```
executes exactly the same code. why copy it?

Reply:
pybullet_envs does not send the parameter `renders=True` to the environments, hence the window is not displayed. I copied the exact same thing from pybullet_envs, but added that additional argument. It is very tightly coupled. I did not find a way to modify the attribute value other than from setting it through env registration.